06-building-a-program-start-to-finish/setting-up-your-repository/output/README.md

```diff
@@ -0,0 +1,8 @@
+#### Output
+- **Using**: GitHub
+- **Deliver**: the repository for your proposed project and at least one `Pull Request`
+- **With**: *independently*
+
+### Link to GitHub Repo
+
+- [My InvertedIndex Repo](https://github.com/andela-fmustapha/inverted-index/ "Visit Faith's Project Repo")
```

Comment:
Did you see the ReadMe guidelines?

I think you should ensure it conforms to that baseline so it's easy to understand what the repo is about.